src/NBomber/Infra/Http.fs

```diff
@@ -24,60 +24,60 @@ let serializeBinary (data: obj) =
     let formatter = BinaryFormatter()
     formatter.Serialize(ms, data)
     ms.ToArray()
-    
-let deserializeBinary<'T>(data: byte[]) = 
+
+let deserializeBinary<'T>(data: byte[]) =
     use ms = new MemoryStream(data)
     let formatter = BinaryFormatter()
-    formatter.Deserialize(ms) :?> 'T 
+    formatter.Deserialize ms :?> 'T
 
-let createHandler (handler: 'TCommand -> 'TResponse) (request: byte[]) = 
-    let command = deserializeBinary<'TCommand>(request)
-    let response = handler(command)
-    serializeBinary(response)   
+let createHandler (handler: 'TCommand -> 'TResponse) (request: byte[]) =
+    let command = deserializeBinary<'TCommand> request
+    let response = handler command
+    serializeBinary response
 
-let sendRequest<'TRequest,'TResponse>(url: Uri) (request: 'TRequest) = async {    
+let sendRequest<'TRequest,'TResponse>(url: Uri) (request: 'TRequest) = async {
     try
         let httpMsg = new HttpRequestMessage(HttpMethod.Post, url)
-        let binaryReq = serializeBinary(request)
+        let binaryReq = serializeBinary request
         httpMsg.Content <- new ByteArrayContent(binaryReq)
-        let! httpResponse = httpClient.SendAsync(httpMsg) |> Async.AwaitTask
+        let! httpResponse = httpClient.SendAsync httpMsg |> Async.AwaitTask

```

Comment:
why to remove here (), it's actually not clear now the invocation flow, I would then rewrite
httpMsg |> client.SendAsync |> Async.Await